src/code/z_sub_s.c

```diff
@@ -259,10 +265,10 @@ s32 SubS_TimePathing_ComputeProgress(f32* progress, s32 elapsedTime, s32 waypoin
 
     *progress = 0.0f;
     if ((waypointTime <= 0) || (elapsedTime < 0)) {
-        return 0;
+        return SUBS_TIME_PATHING_PROGRESS_STATUS_ERROR;
     }
 
-    // When using the knots from `SubS_TimePathing_FillKnots` these nested loops seem to simplify to
+    // When using the knots from SubS_TimePathing_FillKnots() these nested loops seem to simplify to
```

Comment:
If you want this to be a Doxygen comment, you have to use `///` or `//!`. We don't usually bother inside functions unless it's a bug, though.